include/swift/AST/SILOptimizerRequests.h

```diff
@@ -87,6 +87,20 @@ class LoweredSILRequest
                                       ASTLoweringDescriptor desc) const;
 };
 
+class ComputeFieldIndicesRequest :
+    public SimpleRequest<ComputeFieldIndicesRequest,
+                         std::tuple<>(NominalTypeDecl *),
```

Comment:
I think this has to be `void`?

Reply:
IIRC the evaluator doesn't currently support `void` return types, we usually use `evaluator::SideEffect` for that (which is an alias for `std::tuple<>`)

Reply:
Ah interesting, since the definition accepted void I got confused...

Reply:
Yeah the signature written in `SWIFT_REQUEST` isn't currently used for anything so isn't actually checked whether it's consistent or not 😬

Reply:
Yeah I believe the signature written in the `SWIFT_REQUEST` macro isn't actually used for anything for an uncached request, so isn't currently checked for consistency 😬